portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,30 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import java.util.List; 
+import java.util.Arrays; 
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private static final List<String> ANSWERS = 
+      Arrays.asList("A stick!", "A wet log!", "(some) Frogs!");                                                        
 
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+  public void doGet(HttpServletRequest request, HttpServletResponse response) 
+                                                          throws IOException {
+    // Convert the array of answers to JSON
+    String json_answers = convertToJson(ANSWERS);
```

Comment:
lowerCamelCase for the variable name here

Reply:
abf753c.